src/DependencyInjection/Configuration.php

```diff
@@ -47,19 +47,36 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                 ->end()
-                ->arrayNode('stores')
-                    ->normalizeKeys(false)
-                    ->useAttributeAsKey('name')
-                    ->arrayPrototype()
+            ->arrayNode('stores')
+                ->children()
+                    ->arrayNode('chroma_db')
+                        ->children()
+                            ->scalarNode('engine')->defaultValue('chroma-db')->end()
```

Comment:
we could drop the engine key now, right?

Reply:
I think so, yes

Reply:
Or the best with config component 😄 feel free to test my PR or finish it 😊